.github/workflows/ci.yml

```diff
@@ -272,3 +272,26 @@ jobs:
         run: |
           cd ./language/documentation/tutorial/step_1/BasicCoin
           docker run -v `pwd`:/project move/cli build
+
+  generate-documentation:
```

Comment:
@bmwill Good point, we could have something like `ci-pre-land.yml` and ci-post-land.yml` :+1:  However, in this particular case it would lead to reduplication: both, pre-land and post-land must generate the documentation for errors. Only post-land would push the end result to the `gh-pages` branch. I always try to avoid reduplication, but maybe in this case it's unavoidable?

Reply:
I think in this case that would be ok, unless others have any strong opinions on it.